evcache-core/src/main/java/net/spy/memcached/EVCacheMemcachedClient.java

```diff
@@ -86,6 +87,7 @@ public EVCacheMemcachedClient(ConnectionFactory cf, List<InetSocketAddress> addr
         this.appName = client.getAppName();
         this.maxWriteDuration = EVCacheConfig.getInstance().getPropertyRepository().get(appName + ".max.write.duration.metric", Integer.class).orElseGet("evcache.max.write.duration.metric").orElse(50);
         this.maxReadDuration = EVCacheConfig.getInstance().getPropertyRepository().get(appName + ".max.read.duration.metric", Integer.class).orElseGet("evcache.max.read.duration.metric").orElse(20);
+        this.autoLogSwitchOnWrongKey = EVCacheConfig.getInstance().getPropertyRepository().get(appName + ".enable.auto.log.switch", Boolean.class).orElse(false);
```

Comment:
nit: some text to indicate this is for specific for the wrong key error would be helpful, something like "enable.debug.logs.on.wrongkey"

Reply:
Done.